File: Haskell/beginner/1044.hs
-- https://judge.beecrowd.com/pt/problems/view/1044
-- Importing printf
import Text.Printf

-- Main
main = do

    -- Requesting values and assign into variables
    usr_input <- getLine
    let [a, b] = map read (words usr_input) :: [Int]

    -- Finding res
    if a `mod` b == 0 || b `mod` a == 0
        then do
            -- Showing results
            printf "Sao Multiplos\n"
    else do
        -- Showing results
        printf "Nao sao Multiplos\n"

-- By Fernando Serra
-- https://github.com/fernandosserra

File: C/beginner/1044.c
/*https://judge.beecrowd.com/pt/problems/view/1044*/

/*Includes*/
#include <stdio.h>

/*Main Function*/
int main(){
    /*Starting code*/

    /*Variables*/
    int a, b;

    /*Scanning values*/
    scanf("%d %d", &a, &b);

    /*Testing conditions and showing results*/
    if (a % b == 0 || b % a == 0){
        printf("Sao Multiplos\n");
    } else {
        printf("Nao sao Multiplos\n");
    }

    /*Ending*/
    return 0;
}

/*By Fernando Serra*/
/*https://github.com/fernandosserra*/